sdk/test-utils/test-utils/test/tracing/azureTracing.ts

```diff
@@ -0,0 +1,41 @@
+import { useInstrumenter } from "@azure/core-tracing";
+import { assert } from "chai";
+import { TestInstrumenter } from "../../src";
+
+// this is the plugin used in the test file
+function chaiAzureTrace(chai: Chai.ChaiStatic, _utils: Chai.ChaiUtils) {
+  // expect(() => {}).to.supportsTracing() syntax
+  chai.Assertion.addMethod("supportsTracing", function() {
+    this._obj.call(undefined);
+  });
+  // assert.supportsTracing(() => {}) syntax
+  chai.assert.supportsTracing = supportsTracing;
+}
+
+// Implementation
+async function supportsTracing<Callback extends (...args: unknown[]) => unknown>(
+  callback: Callback
+) {
+  const instrumenter = new TestInstrumenter();
+  useInstrumenter(instrumenter);
+  await callback.call(undefined);
+  console.log(callback.name);
+  assert.equal(instrumenter.startedSpans.length, 1);
+}
+
+// types
```

Comment:
Maybe these need to go in a dts file?

Reply:
I think it's actually a little better for them to live in the module scope here. That way you only absorb these declarations if the plugin is loaded.